portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,24 +39,28 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
```

Comment:
There's a comment above that says "TODO:". I think you've made the changes to handle comment data right? If so, you can take out the "TODO:" part of the comment.